cloudinit/sources/helpers/vmware/imc/guestcust_util.py

```diff
@@ -154,4 +154,14 @@ def get_tools_config(section, key, defaultVal):
     return retValue
 
 
+# Sets message to the VMX guestinfo.gc.status property to the
+# underlying VMware Virtualization Platform.
+def set_gc_status(config, gcMsg):
+    if config and config.post_gc_status:
+        rpc = "info-set guestinfo.gc.status %s" % gcMsg
+        return send_rpc(rpc)
+    else:
+        return None
```

Comment:
A nit, but as the above `if` returns unconditionally, we don't need the `else` here:

```suggestion
    return None
```

Reply:
Thanks, the unit test and this comment are addressed. Please check.